README.md

```diff
@@ -154,25 +152,30 @@ Here is a good editor for regexp, maybe try and do it without using it first tho
 ```ruby
 # /User/gollum/precious/ring.rb
 File.dirname(__FILE__)
+
+File.dirname("/User/gollum/precious/")
 ```
 
-- *give me answer*
 
 ```ruby
 # /User/tom/junk/ring.rb
 File.path(__dir__)
+
+File.path("/User/tom/junk/ring.rb")
```

Comment:
`__dir__` is a method on `Kernal` and will return the absolute file path. This was kind of a trick question because `File.path` is redundant here.